jacksonhardaker/adventofcode-2021
# day 14

- generate day
- add readme and input
- initial pass at part 1
- fix part 1 solution
- solve for examples
- resolve call stack overload
- add part 2 to readme
- wire up part 2
- switch array for map
- remove extra loop
- use pattern matching
- add possible solution to part 2
- add comment to test
- remove console log
- use version two of function
- refactor to ensure tail isn't overwritten
- simplify aggregation
- cleanup solution
